cookbook/docs/conf.py

```diff
@@ -137,6 +137,7 @@ def __call__(self, filename):
     "display_version": True,
 }
 
+html_additional_pages = {"index": "index.html"}
 
 examples_dirs = [
```

Comment:
# Material theme options (see theme.conf for more information)
html_theme_options = {

    # Set the name of the project to appear in the navigation.
    'nav_title': 'Flyte',

    # Set you GA account ID to enable tracking
    'google_analytics_account': 'G-YQL24L5CKY',
    # Specify a base_url used to generate sitemap.xml. If not
    # specified, then no sitemap will be built.
    'base_url': 'https://project.github.io/project',

    # Set the color and the accent color
    'color_primary': 'blue',
    'color_accent': 'light-blue',

    # Set the repo location to get a badge with stats
    'repo_url': 'https://github.com/lyft/flyte/',
    'repo_name': 'lyft/flyte',

    # Visible levels of the global TOC; -1 means unlimited
    'globaltoc_depth': 3,
    # If False, expand all TOC entries
    'globaltoc_collapse': False,
    # If True, show hidden TOC entries
    'globaltoc_includehidden': False,
}